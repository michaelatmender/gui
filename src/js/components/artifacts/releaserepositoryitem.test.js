import React from 'react';
import renderer from 'react-test-renderer';
import ReleaseRepositoryItem from './releaserepositoryitem';

it('renders correctly', () => {
  const tree = renderer
    .create(<ReleaseRepositoryItem artifact={{ device_types_compatible: ['test-type'], updates: [], modified: '2019-01-01' }} onExpanded={() => {}} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
