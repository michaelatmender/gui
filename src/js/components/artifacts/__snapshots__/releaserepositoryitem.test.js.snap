// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="release-repo-item flexbox"
>
  <div
    className="muted"
  >
    NaN
  </div>
  <div
    className="MuiPaper-root MuiPaper-elevation1 MuiExpansionPanel-root"
    style={
      Object {
        "border": "1px solid",
        "borderColor": "#e0e0e0",
        "width": "100%",
      }
    }
  >
    <div
      aria-disabled={false}
      aria-expanded={false}
      className="MuiButtonBase-root MuiExpansionPanelSummary-root"
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      style={
        Object {
          "padding": "0 12px",
        }
      }
      tabIndex={0}
    >
      <div
        className="MuiExpansionPanelSummary-content repo-item"
      >
        <div
          aria-describedby={null}
          className="nowrap text-overflow"
          onBlur={[Function]}
          onFocus={[Function]}
          onMouseLeave={[Function]}
          onMouseOver={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          title="test-type"
        >
          test-type
        </div>
        <time
          dateTime="2019-01-01T00:00:00+00:00"
          title={null}
        >
          2019-01-01 00:00
        </time>
        <div
          style={
            Object {
              "maxWidth": "100vw",
            }
          }
        >
          
        </div>
        <div>
          0 Bytes
        </div>
        <button
          className="MuiButtonBase-root MuiIconButton-root expandButton"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 10l5 5 5-5z"
              />
            </svg>
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
    <div
      className="MuiCollapse-container MuiCollapse-hidden"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper"
      >
        <div
          className="MuiCollapse-wrapperInner"
        >
          <div
            role="region"
          >
            <div
              className="MuiExpansionPanelDetails-root"
            >
              <div />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
